Modules/MSCloudLoginAssistant/MSCloudLoginAssistant.psm1

```diff
@@ -159,6 +159,11 @@ function Connect-M365Tenant
         $Global:MSCloudLoginConnectionProfile = New-Object MSCloudLoginConnectionProfile
     }
 
+    if (Compare-InputParametersForChange -CurrentParamSet $PSBoundParameters)
+    {
+        $Global:MSCloudLoginConnectionProfile = $null
```

Comment:
@ykuijs within M365DSC there was a discussion, if this is enough to reset the connection. Shall we otherwise create an new MSCloudLoginConnectionProfile?